samples/nrf9160/nrf_cloud_mqtt_multi_service/src/connection.c

```diff
@@ -656,7 +657,11 @@ static int connect_cloud(void)
 	}
 
 	LOG_INF("Connected to nRF Cloud");
-	return 0;
+	if (IS_ENABLED(CONFIG_NRF_CLOUD_ALERTS)) {
```

Comment:
I made it a library-specific configuration, like CONFIG_NRF_CLOUD_PGPS or CONFIG_NRF_CLOUD_FOTA.  A project's prf.conf can turn off alerts if they don't want them (or simply just not call the one function).

Yes, I should add REST support.  But this is the mqtt multi service sample you are commenting on, so....

Reply:
my point was that applications that use nrf_cloud library (MQTT or REST) shouldn't have to add `IS_ENABLED(CONFIG_NRF_CLOUD_ALERTS)` around all their calls to `nrf_cloud_alert_send`.
its just a different type of cloud message.
i think the kconfig enabling should be changed a bit.

Reply:
Yeah, there is at least some convention in Zephyr of using pre-processor macros to define disabled functions as no-op statements. Maybe we should start doing the same.

Even if we do that, though, in this case I would keep the guard. It would serve no functional purpose, but still be useful to prospective readers since it helps indicate that the function is only used if the feature is enabled.

The guard ***could*** instead be checking an MMSS-specific enable/disable for this feature, but we already use NRF_CLOUD_AGPS (and similar) directly to control whether MMSS uses AGPS (and similar), as Pete points out, so I don't see the harm in doing the same here

Reply:
I'll switch it to being a no-op, and remove the guards around where they are called.  There's no justification for both.